demo_data/organizations.json

```diff
@@ -0,0 +1,48 @@
+{
+  "organizations": [
+    {
+      "users": [
+        {
+          "name": "fjelltopp_member",
+          "capacity": "member"
+        },
+        {
+          "name": "fjelltopp_editor",
+          "capacity": "editor"
+        },
+        {
+          "name": "fjelltopp_admin",
+          "capacity": "admin"
+        }
+      ],
+      "display_name": "Fjelltopp",
+      "description": "Fjelltopp demo organization",
+      "image_display_url": "",
+      "name": "fjelltopp",
+      "state": "active",
+      "title": "Fjelltopp"
+    },
+    {
+      "users": [
+        {
+          "name": "north_pole_member",
+          "capacity": "member"
+        },
+        {
+          "name": "north_pole_editor",
+          "capacity": "editor"
+        },
+        {
+          "name": "north_pole_admin",
+          "capacity": "admin"
+        }
+      ],
+      "display_name": "North Pole",
+      "description": "North Pole demo organization",
+      "image_display_url": "",
```

Comment:
You could just dump a suitable image for MoH north pole at ckanext-unaids/ckanext/unaids/theme/public/img/ and link to it here as with the fjelltopp logo. Maybe [this image](https://upload.wikimedia.org/wikipedia/commons/thumb/d/dd/Coat_of_arms_of_Greenland.svg/2000px-Coat_of_arms_of_Greenland.svg.png )?  

I know it's not essential but it would make me happy. :-)